appveyor.yml

```diff
@@ -56,8 +56,12 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # We don't need to build the example recipe.
-    - cmd: if exist recipes\example rmdir recipes\example /s /q
+    # Find the recipes from master in this PR and remove them.
+    - cmd: cd recipes
+    - cmd: git fetch https://github.com/conda-forge/staged-recipes.git master
+    - cmd: |
+             for /f "tokens=*" %%a in ('git ls-tree --name-only FETCH_HEAD -- .') do rmdir /s /q %%a && echo Removing recipe: %%a
+    - cmd: cd ..
```

Comment:
Popd seems to have non zero exit if not chained. I have had success with lines like 

Pushd something && do op && popd

I'm sure there's a reason for this, but I don't know it.